src/compas/geometry/_intersections/mesh_intersections.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+from compas.geometry import Point
+from compas.geometry import length_vector
+from compas.geometry import subtract_vectors
+from compas.geometry import intersection_line_triangle
+from compas.geometry import intersection_segment_plane
+
+__all__ = [
+    'intersection_mesh_line',
+    'intersection_mesh_plane',
+    'mesh_vertices_to_points',
+]
+
+
+def intersection_mesh_line(mesh, line):
+    """Compute intersection between mesh faces and line. After one single intersection, stops searching for more.
```

Comment:
You are very clear in the docstrings about the behavior of `intersection_mesh_line` and `intersection_mesh_plane`, but I don't like that the one fails early (returning the first point of intersection with a line) and the other gives all points of intersection.  I think the difference should be reflected in the names.

Reply:
rename both to `intersections_mesh_xxx`? with returned intersections sorted based on distance in the case of `line`?